recipes/onnxruntime/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
Is this file used ? It is not exported in conanfile.py (exports_sources) and not used in _cmake.configure

Reply:
Added to export_sources